java-8/appdynamics/README.md

```diff
@@ -8,23 +8,22 @@ Se README for Java 8 for general usage.
 
 ## Configuration of AppDynamics
 
-To enable AppDynamics you must set the environment variable `APPD_ENABLED` to
-true. This can be done in your Dockerfile:
+Basic appdynamics config.
+This can be done in your Dockerfile:
 
 ```
-ENV APPD_ENABLED=true
+ENV APPD_ENABLED true
+ENV APPDYNAMICS_AGENT_ACCOUNT_NAME {PROD | NON-PROD}
+ENV APPDYNAMICS_CONTROLLER_PORT 443
+ENV APPDYNAMICS_CONTROLLER_SSL_ENABLED true
 ```
 
-You must also set `APPDYNAMICS_CONTROLLER_HOST_NAME`. It exists as
-a Fasit property by the name `appdynamics`.
+You must also set host and access key, store them preferably in Vault.
```

Comment:
Disse ligger allerede i Vault og er tilgjengelig for alle apper som har tilgang til Vault.

Reply:
nice